### ISSUE TITLE
[BUG] Network Error

### ISSUE THREAD

**Issue by SyahalTastari** [TiagoSilvaPereira/vemto-issues#33]
```
Error: Network Error
    at e.exports (app://./js/chunk-vendors.ca8bc0b1.js:64:6353)
    at XMLHttpRequest.h.onerror (app://./js/chunk-vendors.ca8bc0b1.js:344:2373)
```

---

vemto 0.5.7
Electron 8.5.2
win32 10.0.18363
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @SyahalTastari could you please give some details about where and how the error happened? Thank you

**Comment by TiagoSilvaPereira** [comment]
Hi @SyahalTastari this issue has fixed on **0.5.10**... Thanks
